cloudapi/v1/cloudapi.proto

```diff
@@ -49,6 +55,25 @@ message V1RepoDetails {
 
 message V2RepoDetails {}
 
+message TaskDetails {
+  string name = 1;
+  string type = 2;
+  string condition_type = 3;
+  string condition_status = 4;
+}
```

Comment:
docstring please ;)

Also, this feels like leaky abstraction. We're not just copying CRD conditions fields over, but instead we're exposing aggregated and high level data to consumers. As a Cloud API consumer, do I really care about the status of each subresource? I think something as simple as showing:

- the status of instance, one of Healthy, Progressing, Degraded, Unknown
- the reason of last transition, e.g., `InstanceTerraform` is updated